src/go/mongolib/proto/main_test.go

```diff
@@ -36,12 +34,11 @@ func TestMain(m *testing.M) {
 	os.Exit(retCode)
 }
 
-func ExampleServerStatus() {
-	ss := proto.ServerStatus{}
-	if err := session.DB("admin").Run(bson.D{{"serverStatus", 1}, {"recordStats", 1}}, &ss); err != nil {
+func ExamplePing() {
+	ss := map[string]interface{}{}
+	if err := session.DB("admin").Run(bson.D{{"ping", 1}}, &ss); err != nil {
 		panic(err)
 	}
 	fmt.Printf("%+v", ss)
-	// Output:
-
+	// Output: map[ok:1]
```

Comment:
@cfsalguero I've changed that test because `serverStatus` gives result depending on machine it was run on. In other words it was giving different result on my local host and travis etc. Let me know if you think it should look different.